### PR TITLE
slightly cleanup mining guides

### PR DESCRIPTION
**Pull request by Sajo8**
- replace img with nicer looking image
- change wordings
- add violetminer to sidebar

### CODE REVIEW

File: docs/guides/mining/violetminer-Guide.md [hunk 1]
```diff
@@ -2,33 +2,37 @@
...
-* [violetminer miner](https://github.com/turtlecoin/violetminer/releases)
+violetminer can be downloaded from its [GitHub Page](https://github.com/turtlecoin/violetminer/releases/latest)
 
 ![violetminer-mining](../../assets/violetminer-mining.png)
 
 ## violetminer Setup and Configuration
 
-* If on Windows, unzip the file into a new folder. There should be an executable, violetminer.exe, in this folder.
-* If on Linux, Mac, or ARM, untar the file into a new folder. (`tar -xf violetminer-linux.tar.gz` for example).
+1. Unzip the file and extract the files into a new folder (Make sure your anti-virus doesn't delete the files)
```
Comment (zpalmtree) [inline docs/guides/mining/violetminer-Guide.md]: Why are we getting rid of more detailed instructions for less detailed instructions?

File: docs/guides/mining/Mining.md [hunk 2]
```diff
@@ -20,17 +20,17 @@ There are currently a few different ways in which you can start mining TurtleCoi
 
 ## Desktop Mining
 
-### [violetminer](https://github.com/turtlecoin/violetminer)
+### [NinjaRig](https://github.com/turtlecoin/ninjarig)
 
-Violetminer currently supports CPUs and Nvidia GPUs in an all in one miner.
+NinjaRig is an all in CPU and GPU miner, modified from xmrig by a TurtleCoin community member.
```
Comment (zpalmtree) [inline docs/guides/mining/Mining.md]: Typo, `NinjaRig is an all in CPU and GPU miner`
Why are we randomly swapping the titles round?

File: docs/guides/mining/NinjaRig-Guide.md [hunk 3]
```diff
@@ -2,17 +2,13 @@
...
 
-NinjaRig can be downloaded from here:
-
-* [NinjaRig Miner](https://github.com/turtlecoin/ninjarig/releases)
+NinjaRig can be downloaded from the [GitHub page](https://github.com/turtlecoin/ninjarig/releases/latest)
 
 ## Downloading and Installing for OS X
 
-Would need to be compiled, instructions don't currently exist...
+Needs to be compiled; instructions are not present as of now, though
```
Comment (zpalmtree) [inline docs/guides/mining/NinjaRig-Guide.md]: Honestly I think the first sentence is easier to understand

File: docs/guides/mining/violetminer-Guide.md [hunk 4]
```diff
@@ -2,33 +2,37 @@
...
 ### Guided Setup
 
 ![guided-setup](../../assets/violetminer-guided-setup.png)
 
 * Double click / run the violetminer executable.
+
 * You will be prompted for a mining pool to use. You can learn more about picking a pool [here](Pools). Make sure you include the pool port. You can find this information on most pools 'Getting Started' page.
-* If you're having difficulty copy/pasting into the command window on Windows, you can click in the top left of the window, then select edit, then paste.
+
+  * If you're having difficulty copy/pasting into the command window on Windows, you can click in the top left of the window, then select edit, then paste.
```
Comment (zpalmtree) [inline docs/guides/mining/violetminer-Guide.md]: What's with the random indentation and spacing after every bullet point?

Comment (ExtraHash) [reply docs/guides/mining/violetminer-Guide.md]: Maybe we could use something like [https://github.com/DavidAnson/markdownlint](markdownlint) to easily enforce a standard formatting? I agree this is not optimal though, unordered lists should not have line breaks in between on markdown.

File: docs/guides/mining/Mining.md (hunk 2 above)
Comment (Sajo8) [reply docs/guides/mining/Mining.md]: i moved them around because ninjarig is probably a better choice for the reader given it also chooses supports AMD, so i moved it up. Both of them are favored over XMRig since you don't need to run separate programs

File: docs/guides/mining/violetminer-Guide.md (hunk 4 above)
Comment (Sajo8) [reply docs/guides/mining/violetminer-Guide.md]: The line breaks makes it easier to read. If there aren't spaces, they're clumped together. Wen you have spaces in between the points, they are spaced apart further and is nicer on the eyes.

File: docs/guides/mining/NinjaRig-Guide.md (hunk 3 above)
Comment (Sajo8) [reply docs/guides/mining/NinjaRig-Guide.md]: How's "needs to be compiled; instructions aren't available yet though". I feel the comma and the "..." are unprofessional/bad english. Doesn't have to be formal language or anything but there should be at least well-formed sentences

File: docs/guides/mining/violetminer-Guide.md (hunk 1 above)
Comment (Sajo8) [reply docs/guides/mining/violetminer-Guide.md]: I think it confuses the reader, but I'm fine with adding the more detailed instructions back

File: docs/guides/mining/violetminer-Guide.md (hunk 4 above)
Comment (ExtraHash) [reply docs/guides/mining/violetminer-Guide.md]: If you want to add padding between bullets, I think that should be handled within the stylesheet

Comment (Sajo8) [reply docs/guides/mining/violetminer-Guide.md]: wat no whyd you do that

Comment (ExtraHash) [reply docs/guides/mining/violetminer-Guide.md]: Wot

Comment (SoreGums) [reply docs/guides/mining/violetminer-Guide.md]: image

if you want to alter formating then alter formatting, not inject a bunch of stupid markup...

File: docs/guides/mining/NinjaRig-Guide.md (hunk 3 above)
Comment (ExtraHash) [reply docs/guides/mining/NinjaRig-Guide.md]: I think both of these sentences are garbage

How about

"Needs to be compiled. There are currently no compilation instructions available."

File: docs/guides/mining/violetminer-Guide.md (hunk 4 above)
Comment (Sajo8) [reply docs/guides/mining/violetminer-Guide.md]: I've replaced it w/ an ordered list, how's that?

File: docs/guides/mining/NinjaRig-Guide.md (hunk 3 above)
Comment (Sajo8) [reply docs/guides/mining/NinjaRig-Guide.md]: Changed text to that

File: docs/guides/mining/violetminer-Guide.md (hunk 1 above)
Comment (Sajo8) [reply docs/guides/mining/violetminer-Guide.md]: added more detailed isntructions back

File: docs/guides/mining/violetminer-Guide.md (hunk 4 above)
Comment (ExtraHash) [reply docs/guides/mining/violetminer-Guide.md]: You need to remove the line breaks, having line breaks in any kind of list is simply incorrect markdown syntax

Comment (Sajo8) [reply docs/guides/mining/violetminer-Guide.md]: Fixed now

File: docs/guides/mining/violetminer-Guide.md [hunk 5]
```diff
@@ -2,34 +2,38 @@
...
+violetminer can be downloaded from its [GitHub Page](https://github.com/turtlecoin/violetminer/releases/latest)
 
 ![violetminer-mining](../../assets/violetminer-mining.png)
 
 ## violetminer Setup and Configuration
 
-* If on Windows, unzip the file into a new folder. There should be an executable, violetminer.exe, in this folder.
-* If on Linux, Mac, or ARM, untar the file into a new folder. (`tar -xf violetminer-linux.tar.gz` for example).
+1. If on Windows, unzip the file into a new folder. There should be an executable, violetminer.exe, in this folder.
+     * If on Linux, Mac, or ARM, untar the file into a new folder. (`tar -xf violetminer-linux.tar.gz` for example).
```
Comment (SoreGums) [inline docs/guides/mining/violetminer-Guide.md]: indent not needed... these items are the same level

File: docs/guides/mining/violetminer-Guide.md [hunk 6]
```diff
@@ -2,34 +2,38 @@
...
+
+4. Next up is the pool password. For most pools, you can just leave this blank, or use `x`. Some pools will use it to allow extra configuration options. Again, check the getting started page for your pool for more information.
+
+5. The next thing to choose is the algorithm. Since you're wanting to mine TurtleCoin, just enter `turtlecoin` here.
+
+6. Finally, we have the rig ID. This can be used if you have multiple computers mining and want to be able to distinguish between them on the pool side. If you don't, just hit the `Enter` key here to leave it blank.
+
+7. You're done! If you want to add another pool, enter `y` on this step. If not, enter `n` and you will start mining if you entered everything correctly!
+
+   * If you want to modify your config, it has been written to the file `config.json`, in the directory you launched violetminer from. You can now just double click violetminer, and it will read from the config file.
```
Comment (SoreGums) [inline docs/guides/mining/violetminer-Guide.md]: indent not needed... this item is not related to the previous one

File: docs/guides/mining/violetminer-Guide.md [hunk 7]
```diff
@@ -2,57 +2,65 @@
...
+4. Next up is the pool password. For most pools, you can just leave this blank, or use `x`. Some pools will use it to allow extra configuration options. Again, check the getting started page for your pool for more information.
+
+5. The next thing to choose is the algorithm. Since you're wanting to mine TurtleCoin, just enter `turtlecoin` here.
 
-Want more information on how to modify the config file? Check out [this section](#config-file-explanation).
+6. Finally, we have the rig ID. This can be used if you have multiple computers mining and want to be able to distinguish between them on the pool side. If you don't, just hit the `Enter` key here to leave it blank.
+
+7. You're done! If you want to add another pool, enter `y` on this step. If not, enter `n` and you will start mining if you entered everything correctly!
+
+You can now just double click violetminer, and it will read from the config file.
```
Comment (zpalmtree) [inline docs/guides/mining/violetminer-Guide.md]: This line makes a little less sense now it's before the explanation that the config has been written to a file - can we reword this or mention that the config has been written?

Something like
```
If you need to relaunch the miner, you don't need to enter all your details again - they've been written to a config file, `config.json`. Just double click violetminer, and it will read all your settings from the config file.

If you want to modify your config, just open up this file - it should be in the same folder as the directory you launched violetminer from.

* Want more information on how to modify the config file? Check out [this section](#config-file-explanation).
```

File: docs/guides/mining/violetminer-Guide.md [hunk 8]
```diff
@@ -2,57 +2,65 @@
...
-* Each `Flag` should be followed by the corresponding `Argument`.
-* For example, since `--pool` takes a `String` argument, we should specify it like so: `violetminer --pool trtl.pool.mine2gether.com:3335`.
-* If a flag takes an argument of `None`, that means you just use the flag, with nothing following it. For example, `violetminer --disableCPU`.
+1. The following tables list all the available command line options.
+
+2. Each `Flag` should be followed by the corresponding `Argument`.
+
+3. For example, since `--pool` takes a `String` argument, we should specify it like so: `violetminer --pool trtl.pool.mine2gether.com:3335`.
+
+4. If a flag takes an argument of `None`, that means you just use the flag, with nothing following it. For example, `violetminer --disableCPU`.
```
Comment (zpalmtree) [inline docs/guides/mining/violetminer-Guide.md]: I think these should remain as bulletpoints, not numbers.

Comment (ExtraHash) [reply docs/guides/mining/violetminer-Guide.md]: Pls stop putting line breaks in between list items

Pls

**Comment by Sajo8** [comment]
Boi

**Comment by brandonlehmann** [comment]
Please correct the deficiencies as listed in the PR comments. When complete, it may be reopened.
